engine/apps/api/views/escalation_policy.py

```diff
@@ -151,3 +161,25 @@ def num_minutes_in_window_options(self, request):
             {"value": choice[0], "display_name": choice[1]} for choice in EscalationPolicy.WEB_DURATION_CHOICES_MINUTES
         ]
         return Response(choices)
+
+    @action(detail=False, methods=["get"])
+    def severity_options(self, request):
+        organization = self.request.auth.organization
+        choices = []
+        if organization.is_grafana_labels_enabled:
+            choices = [
+                {
+                    "value": EscalationPolicy.SEVERITY_SET_FROM_LABEL,
+                    "display_name": EscalationPolicy.SEVERITY_SET_FROM_LABEL_DISPLAY_VALUE,
+                }
+            ]
+        incident_client = IncidentAPIClient(organization.grafana_url, organization.api_token)
+        try:
+            severities, _ = incident_client.get_severities()
```

Comment:
_nit_ I suspect these won't change that often, maybe we can cache these for a few hours?

Reply:
This is only called, once, when editing escalation chains, which is not often either. I think caching wouldn't be necessary in this case, but it is something we have a note to take a look after release :+1:

Reply:
> only called, once, when editing escalation chains

ahh gotcha 👍 (agreed, probably not necessary in that case)